Assets/Treasure/TDK/Runtime/Connect/TDK.Connect.cs

```diff
@@ -175,10 +171,10 @@ public void HideConnectModal()
             TDKConnectUIManager.Instance.Hide();
         }
 
-        public async Task ConnectEmail(string email, Options? options = null)
+        public async Task ConnectEmail(string email)
```

Comment:
Zeeverse keeps the email address in local storage and attempts to silently reconnect on launch because the Thirdweb SDK can do this automatically on iOS and Android. I don't believe we should be removing that feature

Reply:
Added a new commit. With it, they could call `Reconnect(email)` to silently reconnect.